includes/Renderer/Stories/Renderer.php

```diff
@@ -142,12 +155,19 @@ abstract class Renderer implements RenderingInterface, Iterator {
 	 *
 	 * @since 1.5.0
 	 *
-	 * @param Story_Query $query Story_Query instance.
+	 * @param Story_Query $query                  Story_Query instance.
 	 */
 	public function __construct( Story_Query $query ) {
 		$this->query           = $query;
 		$this->attributes      = $this->query->get_story_attributes();
 		$this->content_overlay = $this->attributes['show_title'] || $this->attributes['show_date'] || $this->attributes['show_author'] || $this->attributes['show_excerpt'];
+		// TODO, find a way to inject this a cleaner way.
+		$injector = Services::get_injector();
+		if ( ! method_exists( $injector, 'make' ) ) {
+			return;
+		}
+		$this->assets                  = $injector->make( Assets::class );
+		$this->amp_story_player_assets = $injector->make( AMP_Story_Player_Assets::class );
```

Comment:
This code was written before we had the injector. It is much larger reflector to inject these classes here. Using the injector in the one place, saves doing it lots of and lots of places.